### ISSUE TITLE
Add hover text over the “Ontology Search” button to briefly explain what it does 

### ISSUE THREAD

**Issue by kcs3** [CBIIT/nci-ctd2-dashboard#503]
Compose new text...

**Comment by kcs3** [comment]
For each exactly matched search term, Ontology search traverses the hierarchical ontology structure to return any child nodes that have observations.

**Comment by kcs3** [comment]
Examples where "Ontology Search" provides additional results, as mentioned in #487 (from Zhou):
• for disease context: search "organ", three results. "ontology search", 7 more results
• for ECO: test case: search ECO_0000059 and you will get one result. If you do "ontology search", you will get three more results, which are children of ECO_0000059.
• for observation: ECO_0000059 TP53. there is no direct 'intersection' but there will be intersection between child of ECO_0000059 (i.e. ECO_0000016) and TP53 so at least one observation result

One or more of these could be included in the hover text

**Comment by kcs3** [comment]
Add this modified version of Aris' suggestion as hover text to the Ontology Search button:
"Expand result set to include subjects that are ontology children of original hits".  

Another suggestion, to include the text "Incremental results retrieved through the ontology search are decorated with a "ONTO" icon" is made redundant because we provide a message to that effect above the results.

Add this text to the hover text for the basic search feature:
"Use of quotes to surround multi-word search terms is advised to avoid a large number of hits".

**Comment by kcs3** [comment]
For the final item:
Add this text to the hover text for the basic search feature:
"Use of quotes to surround multi-word search terms is advised to avoid a large number of hits".

the hover text was added to the actual search button rather than to the instructions hover-text over the search box.  Maybe it is good this way...

**Comment by arisf** [comment]
I agree with Ken, the hover text should not be on the Search button. Instead, it should be added to the existing hover text, as follows:
> Please enter the keyword(s) you would like to search on the website. Use of quotes to surround multi-word search terms is advised, to avoid a large number of hits. You may enter multiple search terms, but do not use "AND" or "OR".

**Comment by kcs3** [comment]
Above change tested on columbia test server, verified correct.

**Comment by arisf** [comment]
I see the following behavior (both in Chrome and Edge):
* Submit a search (any will do).
* In the results page, hover over the "Ontology Search" button. This will show the info text. Hover away from the button. This will hide the info text. Up to here, all is fine.
* Now click on the "Ontology Search" button, to initiate an ontology search. This causes the info text to appear again and then remain permanently visible, regardless of hovering or not on the button. Furthermore, it remains visible after navigating to any other Dashboard page.

**Comment by zhouji2013** [comment]
I reproduced this problem. It is an interesting issue. It seems that when Chrome and Edge disabled the button, it becomes totally disabled including the ability to turn off the hover text, but Firefox keeps it able to turn off the hover text.

**Comment by arisf** [comment]
Confirmed on testing instance.

**Comment by arisf** [comment]
tested on production server.
